analyzers/src/SonarAnalyzer.CSharp/Rules/RedundantJumpStatement.cs

```diff
@@ -35,27 +35,27 @@ public sealed class RedundantJumpStatement : SonarDiagnosticAnalyzer
         protected override void Initialize(SonarAnalysisContext context)
         {
             context.RegisterNodeAction(
-                c => CheckForRedundantJumps(c, ((BaseMethodDeclarationSyntax)c.Node).Body),
+                c => CheckForRedundantJumps(c, ((BaseMethodDeclarationSyntax)c.Node)),
```

Comment:
Useless cast, useless parameter
```suggestion
                CheckForRedundantJumps,
```